pkg/netpol/connlist/connlist.go

```diff
@@ -589,8 +630,11 @@ func (ca *ConnlistAnalyzer) getPeersForConnsComputation(pe *eval.PolicyEngine) (
 // if the exposure-analysis option is on, also computes and updates the exposure-analysis results
 func (ca *ConnlistAnalyzer) getConnectionsList(pe *eval.PolicyEngine, ia *ingressanalyzer.IngressAnalyzer) ([]Peer2PeerConnection,
 	[]Peer, error) {
-	// validate focus-direction
-	if err := validateFocusDirectionValue(ca.focusDirection); err != nil {
+	// validate focus-direction and explain-only values
+	if err := ca.validateFocusDirectionValue(); err != nil {
+		return nil, nil, err
+	}
+	if err := ca.validateExplainOnlyValue(); err != nil {
 		return nil, nil, err
 	}
```

Comment:
move to a separate function for flags validation

Reply:
done